packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -74,15 +74,11 @@
 
```

Comment:
This may want to be replaced for many operators? 
This is probably best done in a separate PR in future